portfolio/src/main/webapp/script.js

```diff
@@ -134,4 +133,17 @@ xhttp.onreadystatechange = function() {
   }
 };
 xhttp.open('GET', '/login', true);
-xhttp.send();
\ No newline at end of file
+xhttp.send();
+
+// Load the API key from file and attach to html.
+fetch('./config.json')
+    .then(response => {
+      return response.json();
+    })
+    .then(data => {
+      var script = document.createElement('script');
+      script.src =
+          'https://maps.googleapis.com/maps/api/js?key=' + data.api_key;
+      script.defer = false;
+      document.head.insertBefore(script, document.head.lastChild);
+    });
```

Comment:
Another 4-space problem here...

Reply:
This is the work of clang-format-path. This is how it idents the function because it would be too long for one line.